bin/katello-certs-check

```diff
@@ -175,12 +175,29 @@ function check-ca-bundle-size () {
 
 function check-cert-san () {
     printf "Checking Subject Alt Name on certificate "
-    CHECK=$(openssl x509 -noout -text -in $CERT_FILE | grep DNS:)
+    DNS_LIST=$(openssl x509 -noout -text -in $CERT_FILE | grep DNS:)
     if [[ $? == "0" ]]; then
         success
+        printf "Checking if any Subject Alt Name on certificate matches the Subject CN"
+        SUBJECT_CN=$(openssl x509 -in $CERT_FILE -noout -subject -nameopt multiline|grep commonName|cut -d '=' -f 2|tr -d ' ')
+        for DNS in ${DNS_LIST}
+          do
+          DNS_VALUE="$( echo ${DNS//DNS:/} | tr -d ',')"
```

Comment:
If you want to be correct, it should ignore SANs with the `IP:` prefix and only deal with `DNS:`.

Reply:
This is already the case due to how `DNS_LIST=$(openssl x509 -noout -text -in $CERT_FILE | grep DNS:)` is defined